UHResidentInformationAPI/V1/Infrastructure/ContactLink.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UHResidentInformationAPI.V1.Infrastructure
+{
+    [Table("cccontactlink")]
+    public class ContactLink
+    {
+        [Column("contactno")]
+        [Key]
+        public int ContactID { get; set; }
+
+        [Column("key1")]
```

Comment:
Key2 will be required, which corresponds to person_no from members table

Reply:
Changes added in ca97063